### ISSUE TITLE
Error loading server extension

### ISSUE THREAD

**Issue by mhebrard** [krishnan-r/sparkmonitor#32]
Hi, 
I am using JupyterHub on AWS EMR v5.24
JupyterHub is installed inside a docker container from AWS.
It comes with:
* jupyter_client v5.2.3
* jupyter_core v4.4.0
* jupyterhub v0.9.6

I try to install sparkmonitor within the container as follow

```sh
sudo docker exec jupyterhub bash -c "pip install sparkmonitor"
sudo docker exec jupyterhub bash -c "jupyter nbextension install sparkmonitor --py --user --symlink"
sudo docker exec jupyterhub bash -c "jupyter nbextension enable sparkmonitor --py --user"
sudo docker exec jupyterhub bash -c "jupyter serverextension enable --py --user sparkmonitor"
```

I get message `- Validating: OK` for the two `nbextention` lines but after the last line I get the error message:

```
Enabling: sparkmonitor.serverextension
- Writing config: /home/jovyan/.jupyter
    - Validating...
Error loading server extension sparkmonitor.serverextension
      X is sparkmonitor.serverextension importable?
```

I continue with 

```sh
sudo docker exec jupyterhub bash -c "echo \"c.InteractiveShellApp.extensions.append('sparkmonitor.kernelextension')\" >>  /home/jovyan/.ipython/profile_default/ipython_kernel_config.py"
```

When I run JupyterHub, I see a button "Toggle Spark Monitoring Displays" but I cannot see the monitoring module... any Idea ?

**Comment by piorek** [comment]
Same here.
According to https://github.com/krishnan-r/sparkmonitor/issues/16 running 
```
pip install spark-monitor-s==0.0.11
```
should help but when running this I'm getting
```
pip install spark-monitor-s==0.0.11
Collecting spark-monitor-s==0.0.11
  ERROR: Could not find a version that satisfies the requirement spark-monitor-s==0.0.11 (from versions: none)
ERROR: No matching distribution found for spark-monitor-s==0.0.11
```

**Comment by piorek** [comment]
it turned out it's **sparkonitor-s** not **spark-monitor-s**

**Comment by mhebrard** [comment]
I tried with `sparkmonitor-s` and here the results:

`sudo docker exec jupyterhub bash -c "jupyter nbextension install sparkmonitor --py --user --symlink"`

> Installing /opt/conda/lib/python3.6/site-packages/sparkmonitor/static -> sparkmonitor
> Symlinking: /home/jovyan/.local/share/jupyter/nbextensions/sparkmonitor -> /opt/conda/lib/python3.6/site-packages/sparkmonitor/static
> - Validating: OK
>    To initialize this nbextension in the browser every time the notebook (or other app) loads:
>          jupyter nbextension enable sparkmonitor --user --py

`sudo docker exec jupyterhub bash -c "jupyter nbextension enable sparkmonitor --py --user"`

> Enabling notebook extension sparkmonitor/module...
>      - Validating: OK

`sudo docker exec jupyterhub bash -c "jupyter serverextension enable --py --user sparkmonitor"`

> Enabling: sparkmonitor.serverextension
> - Writing config: /home/jovyan/.jupyter
>     - Validating...
>      sparkmonitor.serverextension  OK

`sudo docker exec jupyterhub bash -c "ipython profile locate default"`

> /home/jovyan/.ipython/profile_default

`sudo docker exec jupyterhub bash -c "echo \"c.InteractiveShellApp.extensions.append('sparkmonitor.kernelextension')\" >>  /home/jovyan/.ipython/profile_default/ipython_kernel_config.py"`

Then I connect to JupyterHub and create a new `PySpark3` notebook

```py
# Init Hail
import hail as hl
hl.init(sc=spark.sparkContext)
```

I get the error: 

> Starting Spark application
>The code failed because of a fatal error:
> 	Session 2 did not start up in 60 seconds..
> 
> Some things to try:
> a) Make sure Spark has enough available resources for Jupyter to create a Spark context.
> b) Contact your Jupyter administrator to make sure the Spark magics library is configured correctly.
> c) Restart the kernel.

**Comment by mhebrard** [comment]
I use sparkmonitor-s and that solve the error previously mentioned.

```sh
sudo docker exec jupyterhub bash -c "pip install sparkmonitor-s==0.0.11"
sudo docker exec jupyterhub bash -c "jupyter nbextension install sparkmonitor --py --user --symlink"
sudo docker exec jupyterhub bash -c "jupyter nbextension enable sparkmonitor --py --user"
sudo docker exec jupyterhub bash -c "jupyter serverextension enable --py --user sparkmonitor"
sudo docker exec jupyterhub bash -c "ipython profile locate default"
sudo docker exec jupyterhub bash -c "echo \"c.InteractiveShellApp.extensions.append('sparkmonitor.kernelextension')\" >>  /home/jovyan/.ipython/profile_default/ipython_kernel_config.py"
```

When I connect to jupyterHub and create a new `PySpark3` notebook, I have a new button in the top menu labelled `Toggle Spark Monitor Display`

But the module do not appears when I run a job...

**Comment by perrital** [comment]
"But the module do not appears when I run a job..."
did you resolve this issue @mhebrard ?

**Comment by dciangot** [comment]
yeah, same here. The problem looks like the import in module.js from the browser debugger:

> Cannot use import statement outside a module modules.js?v=20200513180012:7

**Comment by mhebrard** [comment]
I have some issues with other libraries, so I switch to zeppelin, that match my current needs... I might look back to jupyter later on.
